lib/cb/utils/validator.rb

```diff
@@ -64,6 +63,16 @@ def try_parse_xml(body)
       rescue MultiXml::ParseError
         nil
       end
+
+      def error_message(processed_response)
+        if processed_response['errors']
+          processed_response['errors']
+        elsif processed_response['Errors']
+          processed_response['Errors']
+        else
+          ''
+        end
```

Comment:
How would `fetch` help? I'm not terribly familiar... But the documentation makes no mention of case insensitivity.